File: BookReviewApp-SPA/src/app/_models/book.ts
import { Chapter } from './chapter';
import { Picture } from './picture';

/**
 * @description Interface that represents a Book Entity
 */
export interface Book {
  /**
   * @param id the Id of the Book
   */
  id: number;

  /**
   * @param name the Name of the Book
   */
  name: string;

  /**
   * @param chapters the Chapters of the Book
   */
  chapters: Chapter[];

  /**
   * @param picture the Picture of the Book
   */
  picture: Picture;
}

File: BookReview.API/Models/Book.cs
using System.Collections.Generic;

namespace BookReview.API.Models
{
    public class Book
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public virtual ICollection<Chapter> Chapters { get; set; }
        public virtual Picture Picture { get; set; }
        public virtual ICollection<Read> UsersWhoRead { get; set; }
        
    }
}